server/data/hmppsAuthClient.ts

```diff
@@ -32,26 +31,19 @@ function getSystemClientTokenFromHmppsAuth(username?: string): Promise<superagen
     .timeout(timeoutSpec)
 }
 
-export default class HmppsAuthClient {
-  constructor(private readonly tokenStore: TokenStore) {}
-
-  private static restClient(token: string): RestClient {
-    return new RestClient('HMPPS Auth Client', config.apis.hmppsAuth, token)
-  }
-
-  async getSystemClientToken(username?: string): Promise<string> {
+export default (tokenStore: TokenStore) =>
+  async (username?: string): Promise<string> => {
     const key = username || '%ANONYMOUS%'
 
-    const token = await this.tokenStore.getToken(key)
```

Comment:
The `this` here wasn't working as we were passing the function around so `this` ended up being the global scope instead.  I've therefore switched to the same approach as hmpps prisoner profile for getting the system client token